<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supplier</class>
 <widget class="QDialog" name="Supplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Supplier</string>
  </property>
  <widget class="QLabel" name="supplier_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplier_LineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="contactInformation_GroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Contact information</string>
   </property>
   <widget class="QLabel" name="name_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phoneNumber_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone number : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumber_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_LineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="email_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email : </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="note_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note : </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="note_PlainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>401</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="set_PushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
